polaris.shopify.com/scripts/gen-og-images.ts

```diff
@@ -187,18 +163,19 @@ const genOgImages = async () => {
 
   const getPNG = async (url) => {
     try {
-      const slug = url === '' ? 'home' : url.split('/').pop();
       const imgPath =
-        url.split('/').length > 1 ? url.split('/').slice(0, -1).join('/') : url;
-      const html = await generateHTML(url, slug);
+        url === ''
+          ? 'home'
+          : new URL(url, 'https://polaris.shopify.com').pathname;
+      const html = await generateHTML(typedSiteJSON, url);
       const encodedUrl = `data:text/html;charset=utf-8;base64,${html}`;
       const page = await browser.newPage();
       await page.goto(encodedUrl, {waitUntil: 'networkidle0'});
       const image = await page.screenshot();
       if (!existsSync(`${imgDir}${imgPath}`)) {
         await mkdir(`${imgDir}${imgPath}`, {recursive: true});
       }
-      await writeFile(`${imgDir}${imgPath}/${slug}.png`, image);
+      await writeFile(`${imgDir}${imgPath}.png`, image);
       await page.close();
```

Comment:
Unsure if you want to do this as a part of this PR but if we can:
- Remove the need for chromium to download
- Not need to run puppeteer
- Not need to screenshot a website

To create an og image we would have pretty large win to our CI. I was hoping we could use https://www.npmjs.com/package/@vercel/og which creates an SVG for the og-image. Feel free to do this in this PR or another.

Reply:
We're doing this in a separate PR 👍